bamboo/bootstrap-tf-deployment.sh

```diff
@@ -71,6 +71,7 @@ echo "terraform {
 ../terraform init \
   -input=false
 
+## TODO: This can be removed entirely once most stacks have run through Bamboo
```

Comment:
Why can't it be removed now? Because we have to wait for each deployment to recognize that the resources have been moved?

Reply:
Correct - if we want CI to remove the lambda/IAM resources auto-magically we need to leave it in.    On merge, we'll need to have the team clean this up for their dev stacks if they're not running it regularly.